build/Build.Steps.Windows.cs

```diff
@@ -177,4 +179,31 @@ partial class Build
 
             AssemblyRedirectionSourceGenerator.Generate(netFxAssembliesFolder, generatedSourceFile);
         });
+
+    Target InstallNetFxAssembliesGAC => _ => _
+        .Unlisted()
+        .After(BuildTracer)
+        .OnlyWhenStatic(() => IsWin)
+        .Executes(() => RunNetFxGacOperation(install: true));
+
+    /// <remarks>
+    /// Warning: This target could cause potential harm to your system by removing a required library from GAC.
+    /// </remarks>
+    Target UninstallNetFxAssembliesGAC => _ => _
+        .Unlisted()
```

Comment:
It must have description and be listed😉

Reply:
@pellared should we list everything always or there is some criteria to unlist some targets?